### ISSUE TITLE
star polymer typo in docs (Trac #962)

### ISSUE THREAD

**Issue by RichardHeenan** [SasView/sasmodels#154]
Simple fix, but I cannot build the docs at present on windows, see SasView/sasview#1032, and needs new branch for minor changes to docs. So I am putting in this ticket to remind us to do it.

==========================================
Dear Ziang,
I am not sure if anyone has replied to you previously, if not then apologies we usually respond very quickly.
Thank you for pointing out the typo in the star polymer model documentation. It seems that I was the last person to check this model, so the team can blame me for not noticing. The correction to this will take some while to filter down into a release version of sasview. Personally I would also like to change the fitting parameter in the model from (Rg)^2^ to just Rg (see SasView/sasview#684 ) which would be more logical, but there is some history here.
Thank you again,
Richard Heenan

From: Ziang Li [mailto:lixx3067@umn.edu] 
Sent: 21 April 2017 17:19
To: help@sasview.org
Subject: [Sasview-help] SASView starpolymer model fit typo

Dear !Sir/Madam:

I'm a graduate student from the university of Minnesota and have been using the SASView software for my SAXS data fitting. 

I found a typo in the "help" section of the star polymer model (shape-independent), I think v should equal to u*f/(3f-2) instead of u^2^*f/(3f-2) since u is already defined as u=q^2^*Rg^2^ in the line below. I think it's just a typo in the "help" but it might be worth checking if the code is built according to correct form factor. Thank you!

Best,
Ziang

-- 
Ziang Li
PhD Candidate
Bates and Lodge Research Groups
Department of Chemical Engineering and Materials Science 
University of Minnesota 

Migrated from http://trac.sasview.org/ticket/962
```json
{
    "status": "closed",
    "changetime": "2017-08-27T00:18:52",
    "_ts": "2017-08-27 00:18:52.875332+00:00",
    "description": "\nSimple fix, but I cannot build the docs at present on windows, see #956, and needs new branch for minor changes to docs. So I am putting in this ticket to remind us to do it.\n\n\n==========================================\nDear Ziang,\nI am not sure if anyone has replied to you previously, if not then apologies we usually respond very quickly.\nThank you for pointing out the typo in the star polymer model documentation. It seems that I was the last person to check this model, so the team can blame me for not noticing. The correction to this will take some while to filter down into a release version of sasview. Personally I would also like to change the fitting parameter in the model from (Rg)^2^ to just Rg (see #552 ) which would be more logical, but there is some history here.\nThank you again,\nRichard Heenan\n\nFrom: Ziang Li [mailto:lixx3067@umn.edu] \nSent: 21 April 2017 17:19\nTo: help@sasview.org\nSubject: [Sasview-help] SASView starpolymer model fit typo\n\nDear !Sir/Madam:\n\nI'm a graduate student from the university of Minnesota and have been using the SASView software for my SAXS data fitting. \n\nI found a typo in the \"help\" section of the star polymer model (shape-independent), I think v should equal to u*f/(3f-2) instead of u^2^*f/(3f-2) since u is already defined as u=q^2^*Rg^2^ in the line below. I think it's just a typo in the \"help\" but it might be worth checking if the code is built according to correct form factor. Thank you!\n\nBest,\nZiang\n\n\n-- \nZiang Li\nPhD Candidate\nBates and Lodge Research Groups\nDepartment of Chemical Engineering and Materials Science \nUniversity of Minnesota \n\n\n",
    "reporter": "richardh",
    "cc": "",
    "resolution": "fixed",
    "workpackage": "SasView Bug Fixing",
    "time": "2017-05-16T16:56:07",
    "component": "sasmodels",
    "summary": "star polymer typo in docs",
    "priority": "blocker",
    "keywords": "",
    "milestone": "SasView 4.2.0",
    "owner": "butler",
    "type": "defect"
}
```

**Comment by RichardHeenan** [comment]
Trac update at `2017/05/17 09:28:18`: **richardh** commented:

Further comments from user:  I looked into the code in the "star_polymer.c" file and I think the radius2 defined in the code is actually the Rg^2^ of the entire star polymer rather than the Rg^2^ of a single arm after checking against a related paper (Eqns 1 and 2, Richter, D., et al. "A study of single-arm relaxation in a polystyrene star polymer by neutron spin echo spectroscopy." Macromolecules 22.1 (1989): 468-472.). 

I think the code is consistent with the paper except in the "help" section, the Rg^2^ should be described as the one of the star polymer if the "rg_squared" spilt out in the "Model Parameters" of the software interface is the "double radius2" in the code. Please let me your thoughts after checking. Thank you!

**Comment by butlerpd** [comment]
Trac update at `2017/05/30 03:35:42`: **butler** changed description from:

> 
> Simple fix, but I cannot build the docs at present on windows, see #956, and needs new branch for minor changes to docs. So I am putting in this ticket to remind us to do it.
> 
> 
> ==========================================
> Dear Ziang,
> I am not sure if anyone has replied to you previously, if not then apologies we usually respond very quickly.
> Thank you for pointing out the typo in the star polymer model documentation. It seems that I was the last person to check this model, so the team can blame me for not noticing. The correction to this will take some while to filter down into a release version of sasview. Personally I would also like to change the fitting parameter in the model from (Rg)^2 to just Rg (see #552 ) which would be more logical, but there is some history here.
> Thank you again,
> Richard Heenan
> 
> From: Ziang Li [mailto:lixx3067@umn.edu] 
> Sent: 21 April 2017 17:19
> To: help@sasview.org
> Subject: [Sasview-help] SASView starpolymer model fit typo
> 
> Dear Sir/Madam:
> 
> I'm a graduate student from the university of Minnesota and have been using the SASView software for my SAXS data fitting. 
> 
> I found a typo in the "help" section of the star polymer model (shape-independent), I think v should equal to u*f/(3f-2) instead of u^2*f/(3f-2) since u is already defined as u=q^2*Rg^2 in the line below. I think it's just a typo in the "help" but it might be worth checking if the code is built according to correct form factor. Thank you!
> 
> Best,
> Ziang
> 
> 
> -- 
> Ziang Li
> PhD Candidate
> Bates and Lodge Research Groups
> Department of Chemical Engineering and Materials Science 
> University of Minnesota 
> 
> 
> 

to:

> 
> Simple fix, but I cannot build the docs at present on windows, see #956, and needs new branch for minor changes to docs. So I am putting in this ticket to remind us to do it.
> 
> 
> ==========================================
> Dear Ziang,
> I am not sure if anyone has replied to you previously, if not then apologies we usually respond very quickly.
> Thank you for pointing out the typo in the star polymer model documentation. It seems that I was the last person to check this model, so the team can blame me for not noticing. The correction to this will take some while to filter down into a release version of sasview. Personally I would also like to change the fitting parameter in the model from (Rg)^2^ to just Rg (see #552 ) which would be more logical, but there is some history here.
> Thank you again,
> Richard Heenan
> 
> From: Ziang Li [mailto:lixx3067@umn.edu] 
> Sent: 21 April 2017 17:19
> To: help@sasview.org
> Subject: [Sasview-help] SASView starpolymer model fit typo
> 
> Dear !Sir/Madam:
> 
> I'm a graduate student from the university of Minnesota and have been using the SASView software for my SAXS data fitting. 
> 
> I found a typo in the "help" section of the star polymer model (shape-independent), I think v should equal to u*f/(3f-2) instead of u^2^*f/(3f-2) since u is already defined as u=q^2^*Rg^2^ in the line below. I think it's just a typo in the "help" but it might be worth checking if the code is built according to correct form factor. Thank you!
> 
> Best,
> Ziang
> 
> 
> -- 
> Ziang Li
> PhD Candidate
> Bates and Lodge Research Groups
> Department of Chemical Engineering and Materials Science 
> University of Minnesota 
> 
> 
>

**Comment by butlerpd** [comment]
Trac update at `2017/05/30 16:18:48`: **butler** commented:

Note that this only requires changes in the python model file in sasmodels.  This can currently be edited directly and pushed to master without the need (yet) of a special "docs" branch.

**Comment by butlerpd** [comment]
Trac update at `2017/06/13 21:50:57`: 
* **butler** commented:

Since this would require tagging a version of sasmodels off of the 4.1 tag rather than master this won't be included in 4.1.1 anyway so moving to 4.2 but making blocker so it doesn't get lost given the work to do is trivial.


* **butler** changed milestone from "SasView 4.1.1" to "SasView 4.2.0"
* **butler** changed priority from "trivial" to "blocker"

**Comment by butlerpd** [comment]
Trac update at `2017/08/22 11:50:49`: 
* **butler** commented:

Agree with first point.  fixed that in docs.  Rg however is in fact the arm I think... v/q would be Rg of whole star if I read right .. need to go through math and probably add reference.  Original Benoit reference was incorrect on page number and have updated that in my own local repo.  Will push whole fix to docs once math is checked


* **butler** changed owner from "richardh" to "butler"
* **butler** changed status from "new" to "accepted"

**Comment by butlerpd** [comment]
Trac update at `2017/08/27 00:18:52`: 
* **butler** commented:

In changeset d439007cd436f6eef212f3d956711e6f3d29f2fb:

```text
#!CommitTicketReference repository="sasmodels" revision="d439007cd436f6eef212f3d956711e6f3d29f2fb"
Corrected errors in star_polymer documentation identified by MN student
checked carefully against code, benoit paper and Richter paper cited in
ticket. Also took the opportuinty to generally udated and standardize
the documentation as per ongoing process addresses #106. fixes #154
```


* **butler** changed resolution from "" to "fixed"
* **butler** changed status from "accepted" to "closed"
